packages/dashboard/src/components/Wizard/Wizard.types.ts

```diff
@@ -0,0 +1,71 @@
+import { IStyle } from 'office-ui-fabric-react/lib/Styling';
+import { ISubwayNavStep, SubwayNavStepState } from '../SubwayNav/SubwayNav.types';
+
+export interface IWizardProps {
+  // List of steps in the wizard
+  steps: IWizardStepProps[];
+
+  wizardComplete?: boolean;
+
+  allowSkipAhead?: boolean;
+}
+
+export interface IWizardStepProps {
+  key: string;
+
+  label: string;
+
+  state: SubwayNavStepState;
+
+  disabled?: boolean;
+
+  isSubStep?: boolean;
+
+  onClickStep: (step: ISubwayNavStep, subStep: ISubwayNavStep | undefined) => void;
+
+  wizardContent: IWizardContentProps;
+
+  subSteps?: IWizardStepProps[];
```

Comment:
Because wizard can only have two level.

This can be `Omit<IWizardStepProps, 'subSteps'>`

which in turn saves you a lot of undefined check in runtime.